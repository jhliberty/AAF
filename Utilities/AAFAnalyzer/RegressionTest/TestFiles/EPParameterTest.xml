<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE aaf-file SYSTEM "EPDerivationChain.dtd">
<aaf-file operational-pattern="invalid">
<!-- Make sure all and only valid interpolators are accepted -->
 <top-level name="None Interpolator">
  <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
   <video-speed-control length="0" track-type="picture">
    <varying-speed-ratio interpolator="none" denominator="1" numerator="1"/>
   </video-speed-control>
  </timeline-mob-slot>
 </top-level>
 <top-level name="B-Spline Interpolator">
  <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
   <video-speed-control length="0" track-type="picture">
    <varying-speed-ratio interpolator="b-spline" denominator="1" numerator="1"/>
   </video-speed-control>
  </timeline-mob-slot>
 </top-level>
 <top-level name="Constant Interpolator">
  <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
   <video-speed-control length="0" track-type="picture">
    <varying-speed-ratio interpolator="constant" denominator="1" numerator="1"/>
   </video-speed-control>
  </timeline-mob-slot>
 </top-level>
 <top-level name="Linear Interpolator">
  <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
   <video-speed-control length="0" track-type="picture">
    <varying-speed-ratio interpolator="linear" denominator="1" numerator="1"/>
   </video-speed-control>
  </timeline-mob-slot>
 </top-level>
 <top-level name="Log Interpolator">
  <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
   <video-speed-control length="0" track-type="picture">
    <varying-speed-ratio interpolator="log" denominator="1" numerator="1"/>
   </video-speed-control>
  </timeline-mob-slot>
 </top-level>
 <top-level name="Power Interpolator">
  <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
   <video-speed-control length="0" track-type="picture">
    <varying-speed-ratio interpolator="power" denominator="1" numerator="1"/>
   </video-speed-control>
  </timeline-mob-slot>
 </top-level>
</aaf-file>
