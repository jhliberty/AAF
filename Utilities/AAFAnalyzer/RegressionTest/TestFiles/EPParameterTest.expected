        Name:   Edit Protocol Parameter Test
        Desc:   Verify that all parameters in the AAF file are valid.
        Result: Failed
        Reason: 
        Covered Requirements: REQ_EP_174
        Noted Requirements: None
        Passing Requirements: None
        Info Requirements: None
        Warning Requirements: None
        Failing Requirements: REQ_EP_175

            Name:   Interpolation for Time-Varying Parameters
            Desc:   Interpolation between ControlPoints within VaryingValue objects shall be specified as one of the following: ConstantInterpolator, LinearInterpolator, LogInterpolator, PowerInterpolator, BSplineInterpolator.
            Node:   6 (VaryingValue)
            Result: Failed
            Reason: VaryingValue object in Mob slot with ID = 1 of Top-Level Composition "None Interpolator" has an illegal interpolation definition.
            Covered Requirements: None
            Noted Requirements: None
            Passing Requirements: None
            Info Requirements: None
            Warning Requirements: None
            Failing Requirements: REQ_EP_175
            Detail: AUID = 5b6c85a3-0ede-11d3-80a9006008143e6f).

